index.html

```diff
@@ -1377,22 +1377,22 @@ <h4 id="example-get-all-patient-types">Example: Get Payment Profiles By Customer
             <pre class="center-column">
 [ 
     {
-        profileName : profileName,
+        profileName : 'profileName',
         isDefault : true,
-        cardNumberLast4 : cardNumberLast4,
-        customerId : customerId,
-        id : 0,
-        expiry : expiry,
-        creditCardType : creditCardType     
+        cardNumberLast4 : 'cardNumberLast4',
+        customerId : 'customerId',
```

Comment:
'customerId' should be the same as sent on URL (line 1370 : 
"GET https://payfac.nextech.com/api/public/paymentProfiles/1738")

Reply:
done